packages/node/src/integrations/tracing/google-genai/index.ts

```diff
@@ -0,0 +1,74 @@
+import type { GoogleGenAIOptions, IntegrationFn } from '@sentry/core';
+import { defineIntegration, GOOGLE_GENAI_INTEGRATION_NAME } from '@sentry/core';
+import { generateInstrumentOnce } from '@sentry/node-core';
+import { SentryGoogleGenAiInstrumentation } from './instrumentation';
+
+export const instrumentGoogleGenAI = generateInstrumentOnce(
+  GOOGLE_GENAI_INTEGRATION_NAME,
+  () => new SentryGoogleGenAiInstrumentation({}),
+);
+
+const _googleGenAIIntegration = ((options: GoogleGenAIOptions = {}) => {
+  return {
+    name: GOOGLE_GENAI_INTEGRATION_NAME,
+    options,
```

Comment:
l/m: Not a super strong opinion, but IMHO it would be nicer if we could do without this. It would be nicer if we could just pass the options to the `instrumentGoogleGenAI(options)` method and use it from there, instead of looking this up from the integration like this. That's not an ideal pattern, it kind of works but couples things together in a way that we should only do if there is really no other way to do that. Looking through where we use it, it is really mostly two places, `getRecordingOptionsFromIntegration`,  where the options can already also be passed in directly - let's just make this mandatory and always use the passed in options, maybe? - and `_patch`, where we could pass it in as instrumentation config in the constructor.

Reply:
putting options etc. on the integration (basically any non-hook) should really be a last-resort thing that we should avoid whenever possible, as this is then public API and can't be changed anymore easily etc 😅